### PR TITLE
Objects Automated Test - LPS-156562 | master

### PR DESCRIPTION
**Pull request by Italo-gabriel**
[LPS-156562](https://issues.liferay.com/browse/LPS-156562)

### CODE REVIEW

**Comment by Italo-gabriel** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -385,4 +385,25 @@ definition {
 			actionName = "Update Action Name");
 	}
 
+	@description = "LPS-146871 - Verify that the Action name is required"
+	@priority = "4"
+	test VerifyActionNameIsRequired {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 1",
+			objectName = "CustomObject1",
+			pluralLabelName = "Custom Objects 1");
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:394]: It looks like the file is using an older naming convention. For now, I'm going to enforce the using the ticket number until we decide on a different approach.

**Review by timpak — APPROVED** [review]
Hey @Italo-gabriel, good job with this test! I made a small change here: c53abcb0e47a

**Comment by timpak** [comment]
Added to https://github.com/timpak/liferay-portal/pull/330

**Pull request closed without merging**